Dockerfiles/midstream/unit_tests.py

```diff
@@ -3,78 +3,72 @@
+        
 
     # Running the container with correctly set environment variables, default channel missing, all tasks should pass
     def test_positive_missing_default_channel(self):
-        exec_cmd = """
-        export IMAGE_TO_TEST=quay.io/cvpops/test-operator:missing-default-channel-v1
-        /run_tests.py
-        """
-        result = subprocess.run(exec_cmd, shell=True)
-        self.assertFalse(os.path.exists(".errormessage"), "File .errormessage should not be present!")
+        self.env["IMAGE_TO_TEST"] = "quay.io/cvpops/test-operator:missing-default-channel-v1"
+        result = subprocess.run(self.exec_cmd,
+                                shell=True,
+                                env=self.env,
+                                cwd=SCRIPT_DIR)
         self.assertEqual(0, result.returncode)
 
     # Set env variable to custom made image that makes playbook extract-operator-bundle.yml fail
     # Due to parsing error
     def test_negative_parsing(self):
-        exec_cmd = """
-        export IMAGE_TO_TEST=quay.io/cvpops/test-operator:missing-alm-examples-v1
-        /run_tests.py
-        """
-        result = subprocess.run(exec_cmd, shell=True)
-        with open(".errormessage") as error_file:
+        self.env["IMAGE_TO_TEST"] = "quay.io/cvpops/test-operator:missing-alm-examples-v1"
+        result = subprocess.run(self.exec_cmd,
+                                shell=True,
+                                env=self.env,
+                                cwd=SCRIPT_DIR)
+        with open(SCRIPT_DIR+".errormessage") as error_file:
             self.assertIn("Result code:", error_file.read(), "Result code not found in %s" % error_file)
         self.assertEqual(50, result.returncode)
-        os.remove(".errormessage")
 
     # Set env variable to custom made image that makes validation fails with following error:
     # ERRO[0003] error validating format in /tmp/bundle-688328851: Bundle validation errors: couldn't
     # parse dependency of type olm.crd
     def test_negative_image_bundle_validation(self):
-        exec_cmd = """
-        export IMAGE_TO_TEST=quay.io/cvpops/test-operator:invalid-dependencies-v1
-        /run_tests.py
-        """
-        result = subprocess.run(exec_cmd, shell=True)
-        with open("/project/output/validation-output.txt") as error_file:
+        self.env["IMAGE_TO_TEST"] = "quay.io/cvpops/test-operator:invalid-dependencies-v1"
+        result = subprocess.run(self.exec_cmd,
+                                shell=True,
+                                env=self.env,
+                                cwd=SCRIPT_DIR)
+        with open(SCRIPT_DIR+"/output/validation-output.txt") as error_file:
             self.assertIn("Bundle validation errors: couldn't parse dependency of type olm.crd", error_file.read(), "Result code not found in %s" % error_file)
         self.assertEqual(70, result.returncode)
-        os.remove("/project/output/validation-output.txt")
```

Comment:
Why it was removed? Each test should have clean "env" and leftover from previous test shoudl not cause failure to other test.

Reply:
updated